### ISSUE TITLE
Feature request: connection data population from clipboard on new connection dialog opening in NetRocks

### ISSUE THREAD

**Issue by MaxD2** [elfmz/far2l#1156]
По работе часто приходится подключаться к FTP клиентов, данные о которых присылают по почте. Было бы невероятно круто, если бы при открытии окна нового подключения в NetRocks анализировался буфер обмена и если в нем содержатся параметры подключения - они бы прописывались в диалоговое окно.

На базовом уровне - это поддержка ссылок вида ftp://user:pass@host/

На крутом - всякого человеческого формата типа
ftp.server.com
user - max
pass - secret

Не будет большой беды, если что-распознается неправильно, так как это все видно в диалоговом окне и можно сразу подправить.

**Comment by elfmz** [comment]
если что нетрокс умеет открывать подключения сохраненные в отдельных конфигах, формат конфига можно подсмотреть экспортнув в какую нить директорию какой нить готовое подключение с помощью F5

**Comment by MaxD2** [comment]
Да, это хорошая функция. Но как ее использовать для вышеозвученого без добавления дополнительных телодвижений в процесс?

**Comment by elfmz** [comment]
без телодвижений никак( насчет клипборда подумаю, в принципе реально, хотя и несколько специфическая фича

**Comment by MaxD2** [comment]
Спасибо! 

Мне кажется, что фича не сильно специфичная - данные подключения нечасто набираются из памяти руками, в большинстве случаев они откуда-то копируются за несколько переключений туда-сюда. Многие будут рады, если это все будет выцепляться автоматом. Единственное, что надо будет как-то донести, что так можно )
